app/controllers/import_rdf_controller.rb

```diff
@@ -0,0 +1,39 @@
+require 'json/ld'
```

Comment:
Is this necessary? If it IS, can we move it somewhere more central? Application.rb? Environment.rb?